recipes/billingegroup-meta/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "billingegroup-meta" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  skip: True  # [py36 or win]
+  number: 0
+
+requirements:
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - seaborn
+    - fabio
+    - pims
+    - yaml
+    - pandas
+    - jupyter
+    - pyFAI  # [linux]
+
+test:
+  imports:
+    - matplotlib
+    - pyFAI
```

Comment:
You should add a selector on pyfai here too since it will only be available on linux. I am curious why you picked matplotlib and pyfai to import from the 10 listed runtime dependencies. You might consider importing *all* of the listed runtime dependencies (other than python, of course), just to make sure that they are all actually importable.